const User = require('../models/User')

module.exports.register = async (req, res) => {
    try {
        
        console.log("ğŸš€ ~ register here: ", req.body)
        const userCreated = await User.create( req.body)
        console.log("ğŸš€ ~ userCreated", userCreated)
        res.send({success: true})
    } catch (error) {
    
        console.log("ğŸš€ ~ Error in register", error.message)

        res.send({success: false, error: error.message})
        
    }
}

module.exports.list = async (req, res) => {

    try {
        
        console.log("ğŸš€ ~ list here: ")

        const users = await User.find()
        console.log("ğŸš€ ~ users", users)
       
        res.send({success: true, users})
    } catch (error) {
    
        console.log("ğŸš€ ~ Error in list users", error.message)

        res.send({success: false, error: error.message})
        
    }


}